packages/compass-e2e-tests/helpers/commands/hide-toasts.ts

```diff
@@ -0,0 +1,21 @@
+import type { CompassBrowser } from '../compass-browser';
+import * as Selectors from '../selectors';
+
+export async function hideToasts(browser: CompassBrowser): Promise<void> {
```

Comment:
Super nit on the naming

```suggestion
export async function hideAllVisibleToasts(browser: CompassBrowser): Promise<void> {
```